openlibrary/i18n/te/messages.po

```diff
@@ -275,78 +275,78 @@ msgstr "మీ ప్రస్తుత సంకేతపదం"
 
 #: borrow.html:85
 msgid "Loan Expires"
-msgstr ""
+msgstr "రుణం గడువు"
 
 #: borrow.html:86
 #, fuzzy
```

Comment:
@jnanadattasingaraju Where there is this attribute `fuzzy`, can you review the msgstr and if it looks accurate, remove the `fuzzy`? These are often created by machine matching, so may be close, or completely wrong. Even if they are correct, they still need a review and removal of the `fuzzy` tag before they will show up on the live site.

We should create some guidelines for translators and put this info there, thank you for being an early participant! :smile: 

For reference: documentation describing `fuzzy` https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html